Formula/par.rb

```diff
@@ -15,7 +15,7 @@ class Par < Formula
 
   conflicts_with "rancid", because: "both install `par` binaries"
 
-  # A patch by Jérôme Pouiller that adds support for multibyte
+  # A patch by Jerome Pouiller that adds support for multibyte
```

Comment:
If ASCII-only is truly unavoidable, I think we should abbreviate the first name as “J.” instead of mutilating it by stripping those accents.